Lib/pathlib.py

```diff
@@ -583,6 +583,8 @@ def __len__(self):
             return len(self._parts)
 
     def __getitem__(self, idx):
+        if isinstance(idx, slice):
+            return [self[i] for i in range(*idx.indices(len(self)))]
```

Comment:
What about use `self[_] for _ in range ...`

Reply:
I believe from experiences in the Python world that using underscore as a variable name implies you won't be using it at all.